cmd/argoexec/commands/root.go

```diff
@@ -129,11 +129,11 @@ func initExecutor() *executor.WorkflowExecutor {
 		progressFileTickDuration,
 	)
 
+	log.Debugf("template: %s", wfv1.MustMarshallJSON(&wfExecutor.Template))
```

Comment:
Agreed. We probably don’t need this log. Consider removing it completely or just printing of template name.